File: Java/Contrato.java
public class Contrato implements Interface_Texto{

    private String assinante;
    private double valor_contrato;

    public Contrato(String assinante,double valor_contrato){
        this.assinante = assinante;
        this.valor_contrato = valor_contrato;
    }

    @Override
    public void mostrarTexto() {
        System.out.println("Assinante do contrato: "+this.assinante);
        System.out.println("Valor do contrato: "+this.assinante);
    }
}

File: Python/Contrato.py
import InterfaceTexto

class Contrato(InterfaceTexto):
    def __init__(self, assinante, valor_contrato):
        self.assinante = assinante
        self.valor_contrato = valor_contrato

    def mostrar_texto(self):
        print(f"Assinante do contrato: {self.assinante}")
        print(f"Valor do contrato: R$ {self.valor_contrato}")